xhtml2pdf/tags.py

```diff
@@ -100,7 +100,7 @@ def start(self, c):
 
 
 class pisaTagA(pisaTag):
-    rxLink = re.compile("^(#|[a-z]+\:).*")
+    rxLink = re.compile(r"^(#|[a-z]+\:).*")
```

Comment:
I guest this can be like,

`rxLink = r"^(#|[a-z]+\:).*"`

Reply:
I'm not too much into regex yet, but I think that that's possible.